meteor/server/publications/rundown.ts

```diff
@@ -367,9 +375,29 @@ meteorPublish(
 		return null
 	}
 )
+
+meteorPublish(
+	PeripheralDevicePubSub.expectedPlayoutItemsForDevice,
+	async function (deviceId: PeripheralDeviceId, token: string | undefined) {
+		if (!deviceId) throw new Meteor.Error(400, 'deviceId argument missing')
```

Comment:
Should this publication use a better authentication? Like in `PeripheralDevicePubSub.rundownsForDevice`?

https://github.com/nrkno/sofie-core/pull/1056/files#diff-0a1e64f58116bd28a7682f7327410f9d24631a4b0c877257e54eeecb917cb4f6R59